astropy/table/tests/test_pprint.py

```diff
@@ -564,7 +563,7 @@ def test_pprint_py3_bytes():
     is printed correctly (without the "b" prefix like b'string').
     """
     val = bytes('val', encoding='utf-8')
-    blah = 'bläh'.encode('utf-8')
+    blah = 'bläh'.encode()
```

Comment:
First example where I feel there is a bit of loss of clarity. Though happy to let it pass just so that we do not get to see it again...

Reply:
I agree on both points.